db/fixtures/tactical_vote_stt_recs.rb

```diff
@@ -31,31 +41,48 @@ def load
       rec = Recommendation.find_or_initialize_by(rec_key)
 
       source_advice = row[:advice]
-      advice_is_not_party = ACCEPTABLE_NON_PARTY_ADVICE.include?(source_advice)
 
-      if advice_is_not_party
-        rec.text = ADVICE_TRANSLATION[source_advice] || source_advice
-        # rubocop:disable Lint/UselessAssignment
-        party_short_code = nil
-        acceptable = !rec.text.nil?
-      else
-        rec.text = source_advice
-        party_short_code = rec.party_short_code_from_text
-        acceptable = !party_short_code.nil?
-      end
+      # ------------------------------------------------------------------------
 
-      if rec.text == "" || !acceptable
+      canonical_advice = source_advice.downcase.parameterize(separator: "_").to_sym
+      party_smv_code = SMV_CODES_BY_RECOMMENDATION_TEXT[canonical_advice]
+      non_party_advice = if party_smv_code.nil?
+                           ACCEPTABLE_NON_PARTY_ADVICE.include?(canonical_advice) ? canonical_advice : nil
+                          else
+                            nil
+                          end
+
+      if party_smv_code && parties_by_smv_code[party_smv_code]
+        rec.text = party_smv_code.to_s.titleize
```

Comment:
Why not use `party.name` from the `Party` model?  Wouldn't that ensure that party names are displayed consistently everywhere?

Reply:
The text property of the Recommendation is presently displayed in the recommendations table which does not have much horizontal space, hence the use of (mostly) 3 letter codes.

Regarding uniqueness, using the party name creates an unwelcome linkage between UI and code. If someone should decide the name shown on the UI should be 'Conservative' instead of 'Conservatives' then any code that refers to 'Conservatives' has to change.